kubernetes/client/api/custom_objects_api.py

```diff
@@ -93,6 +93,7 @@ def create_cluster_custom_object_with_http_info(self, group, version, plural, bo
                                  number provided, it will be total request
                                  timeout. It can also be a pair (tuple) of
                                  (connection, read) timeouts.
+        :param _response_type: class for deserialized object.
```

Comment:
kubernetes/client/api/custom_objects_api.py:  the file is generated, any change here would be overwritten by the  generation of the file next time. please make changes in the generator, https://github.com/openapitools/openapi-generator